serverlessworkflow/modules/ROOT/pages/cloud/operator/referencing-resource-files.adoc

```diff
@@ -14,6 +14,15 @@ For example, when doing xref:service-orchestration/orchestration-of-openapi-base
 
 If these files are not in a remote location that can be accessed via the HTTP protocol, you must describe in the `SonataFlow` CR where to find them within the cluster. This is done via link:{kubernetes_configmap_url}[`ConfigMaps`].
 
+[IMPORTANT]
+====
+The `.spec.resources` section is **ignored in the `preview` profile**.
```

Comment:
@ricardozanini shouldn't it be ignored only in `gitops` profile, but respected in `preview` and `dev` when the resources are required to build the image?